forward43/scraper_masterpeace.py

```diff
@@ -0,0 +1,222 @@
+"""Scaper for Masterpeace projects."""
+from typing import Dict, List
+
+from surveymonkey import client
+from forward43.scraper import ForwardScraper
+
+def get_client(token):
+    """Get surveymonkey client."""
+    surveymonkey_client = client.Client(access_token=token)
+
+    return surveymonkey_client
+
+class MasterpeaceScraper(ForwardScraper):
+    """Scrapes Masterpeace clubs' reports of projects."""
+    def __init__(self, token: str):
+        ForwardScraper.__init__(self, 'masterpeace')
+
+        self.client = get_client(token)
+
+    def get_survey_responses(self, search_string: str) -> List[Dict]:
+        """Get the correct surveys for Masterpeace projects."""
+        surveys = self.client.get_survey_lists()
+        if 'data' not in surveys:
+            raise ValueError(
+                f"Surveys not found for search string {search_string}, data returned: {surveys}"
+            )
+
+        scrapable_surveys = {}
+        for survey in surveys["data"]:
+            if search_string in survey["title"]:
+                scrapable_surveys[survey["id"]] = survey
+
+        if len(scrapable_surveys) == 0:
+            raise ValueError(
+                f"Surveys not found for search string {search_string}, data returned: {surveys}"
+            )
+
+        for survey_id in scrapable_surveys.keys():
+            responses_dict = {}
+            responses = self.client.get_all_pages_response(survey_id)
+            for response in responses:
+                if not response.get("data", []):
+                    raise ValueError(
```

Comment:
This will throw an error for the first item in the list that does not have `data` param. Is this the intended behaviour?

If not, you could do something like this:
```
for response in responses:
    data = response.get('data', [])

    for response_data in data:
        respondent_id = response_data.get("id", "")
        responses_dict[survey_id][respondent_id] = response_data
```